tests/test_mcmc_helper.py

```diff
@@ -52,18 +53,31 @@ def test_mcmc_helper_target_function():
 
     target_log_prob_fn = hmc_helper.target_log_prob_fn
 
-    assert model.log_marginal_likelihood() == target_log_prob_fn()
+    def compute_jacobian_correction_term():
+        correction_term = tf.constant(0., dtype=default_float())
+        for param in model.trainable_parameters:
+            if param.transform is not None:
+                value = param.read_value()
+                log_det_jacobian = param.transform.forward_log_det_jacobian(value, value.shape.ndims)
+                correction_term += tf.reduce_sum(log_det_jacobian)
+        return correction_term
```

Comment:
This is just the same code again as in mcmc.py - what could a test look like that actually checks *behaviour*?

Reply:
Yep agreed, the tests are now updated!